phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -94,6 +99,8 @@ public void testCreateTable() throws Exception {
                 "                id INTEGER not null primary key desc\n" + 
                 "                ) ";
         conn.createStatement().execute(ddl);
+        HBaseAdmin admin = driver.getConnectionQueryServices(getUrl(), props).getAdmin();
+        assertNotEquals(null, admin.getTableDescriptor(Bytes.toBytes(tableName)));
```

Comment:
Minor nits: assertNotNull reads better here. Same recommendations as above regarding getting and closing HBaseAdmin.